src/api/app/services/meta_controller_service/meta_validator.rb

```diff
@@ -0,0 +1,22 @@
+module MetaControllerService
+  class MetaValidator
+    require_dependency 'opensuse/validator'
+
+    attr_reader :meta, :request_data, :errors
+
+    def initialize(params = {})
+      @meta = params[:meta] if params.key?(:meta)
```

Comment:
The `if` is not needed because if the key `meta` is not present in the `params` hash, it will return `nil`, which what the value of the instance variable `@meta` already is.